### PR TITLE
Change database.yml.sample to postgresql

### PR DESCRIPTION
**Pull request by lucasmoura**
The setup script is getting by default the sqlite3 database config file. However, the project does not work properly on sqlite3 anymore. Therefore, sqlite3 should no longer be the default database when running the setup script. Therefore, this merge request change the default database to postgresql.

@GustavoCayres, @Herez

### CODE REVIEW

**Comment by rafamanzo** [comment]
It just struck me: how was Travis' build succeding with SQLite 3?

**Comment by lucasmoura** [comment]
Travis was not using the SQLite 3 database. It was using the database.yml.postgresql_sample instead of the database.yml.sample.

**Comment by rafamanzo** [comment]
Can you ass something like this to the changelog? `Drop SQLite 3 support`

After that I believe we're ready to merge. Nice job!

**Comment by lucasmoura** [comment]
Done :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ KalibroConfigurations is the configuration storage web service for Mezuro.
 ## Unreleased
 
 - Disable rubocop Lint/HandleExceptions cop for rake
+- Drop SQLite 3 support
```
Comment (rafamanzo) [inline CHANGELOG.md]: Next time, place it at the top :)

Don't worry about this one.

**Pull request merged**